apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -204,12 +204,29 @@ function trackBlocksHandler( blocks, eventName, propertiesHandler = noop, parent
  * @param {string} eventName event name
  * @returns {Function} track handler
  */
-const getBlocksTracker = ( eventName ) => ( blockIds ) => {
+const getBlocksTracker = ( eventName ) => ( blockIds, fromRootClientId, toRootClientId ) => {
 	const blockIdArray = Array.isArray( blockIds ) ? blockIds : [ blockIds ];
 
+	const fromContext = maybeAddBlockEventContext( fromRootClientId );
+	const toContext = maybeAddBlockEventContext( toRootClientId );
+
+	let context;
+	let from_context;
+	let to_context;
+	if ( toRootClientId === undefined || fromContext === toContext ) {
+		context = fromContext;
+	} else {
+		from_context = fromContext;
+		to_context = toContext;
+	}
```

Comment:
Technically this extra complexity for from/to is only valid for tracking the `moveBlockToPosition` action.  Currently this is supposed to track reusable blocks but is not (see issue linked in PR description).  However, this event is being fired on rare occasions by another means so may be worth keeping anyways.

Reply:
I looks like the block selection button is this which is found in select mode:

![Screen Shot 2021-07-09 at 12 42 42 PM](https://user-images.githubusercontent.com/28742426/125110858-29346700-e0b3-11eb-86c9-46569ba2a183.png)

Im not sure how to get its `moveBlocksToPosition` action to fire.